interoperator/pkg/watches/watches_test.go

```diff
@@ -31,8 +30,6 @@ var kubeConfig *rest.Config
 var sch *runtime.Scheme
 var c client.Client
 
-const timeout = time.Second * 5
```

Comment:
Surprised to see so many unused variables. I always thought go by default points this out when compiling.

Reply:
yes! this was surprising!